pkg/sdk/accounts_integration_test.go

```diff
@@ -96,6 +107,32 @@ func TestInt_AccountCreate(t *testing.T) {
 		require.NoError(t, err)
 		_, err = client.Accounts.ShowByID(ctx, newAccountID)
 		require.NoError(t, err)
+
+		// drop account
+		err = client.Accounts.Drop(ctx, newAccountID, 3, &DropAccountOptions{
+			IfExists: Bool(true),
+		})
+		require.NoError(t, err)
+
+		// check if account is dropped
+		_, err = client.Accounts.ShowByID(ctx, newAccountID)
+		require.Error(t, err)
+
+		// undrop account
+		err = client.Accounts.Undrop(ctx, newAccountID)
+		require.NoError(t, err)
+
+		// check if account is undropped
+		_, err = client.Accounts.ShowByID(ctx, newAccountID)
+		require.NoError(t, err)
+
+		// drop account again
```

Comment:
why are we dropping it again?

Reply:
create -> drop -> undrop -> drop again

its just a final cleanup.